abstract_test.go

```diff
@@ -443,17 +438,17 @@ func TestResolveTypeOnInterfaceYieldsUsefulError(t *testing.T) {
 				nil,
 			},
 		},
-		Errors: []graphqlerrors.GraphQLFormattedError{
-			graphqlerrors.GraphQLFormattedError{
+		Errors: []FormattedError{
```

Comment:
@lenaten I think we should go for `ErrFormatted`, is a Go convention to use `Err` suffix for custom errors.